ui/shared/ad/CustomAdBanner.tsx

```diff
@@ -0,0 +1,48 @@
+import { Flex, chakra, Tooltip, Image } from '@chakra-ui/react';
+import React, { useState, useEffect } from 'react';
+
+import useCustomBanners from 'ui/shared/ad/useCustomBanners';
+
+const CustomAdBanner = ({ className }: { className?: string }) => {
+  const customBanners = useCustomBanners().data || [];
```

Comment:
As a side note, we don't need to prepend variables with `custom`-word, since this code is already inside the `CustomAdBanner.tsx` file. So, it is a little bit wordy now.

Reply:
Ok, I'll rename it.